src/components/formly.form.ts

```diff
@@ -24,7 +24,7 @@ export class FormlyForm implements OnInit  {
 
   constructor(
     private formlyConfig: FormlyConfig,
-    private formBuilder: FormBuilder
+    private formBuilder: FormBuilder,
```

Comment:
Trailing comma's have become a practice in the industry, so that you do not have two line changes.

I would keep this change, in fact we can make this mandatory in tsconfig as well